nucypher/characters.py

```diff
@@ -434,15 +442,34 @@ def public_key(self, power_up_class: ClassVar):
         power_up = self._crypto_power.power_ups(power_up_class)
         return power_up.public_key()
 
+    @property
+    def public_address(self):
+        # TODO: Figure out the *real* way to cast addresses to bytes (ie, hex or whatever without violating checksum).
```

Comment:
https://github.com/ethereum/eth-utils#to_canonical_addressvalue---bytes

Reply:
OK, now I understand - I gave `to_bytes` a spin and didn't like it, but I didn't realize that I had inadvertently rebuilt `to_canonical_address` :-)